### ISSUE TITLE
Update link in Github config

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-pride#12]
**Update Github Config Link**
The link in the `.github/config.yml` is incomplete. Please update the link to include the `.html` extension at the end of the url. 

It should look like this:

``` yml
We'd love to celebrate you and your contribution to the community. Let us say "Thank You" again, by sending you a [special gift](https://legesher.io/dreamteam.html) only given to special individuals like you who have chosen to give back.
```

**Comment by mikepetridisz** [comment]
@madiedgar created a PR, can we merge it?

**Comment by madiedgar** [comment]
Absolutely @mikepetridisz! I'll take a look at it now 👍

**Comment by madiedgar** [comment]
Closing this as @mikepetridisz solved it in #13 🎉
